security/ima-evm-utils-tests.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2022 IBM
+# Author: Nageswara R Sastry <rnsastry@linux.vnet.ibm.com>
+
+import os
+from avocado import Test
+from avocado.utils import build, distro, git, process
+from avocado.utils.software_manager.manager import SoftwareManager
+
+
+class IMAEVMUtils(Test):
+    """
+    ima-evm-utils testsuite
+    :avocado: tags=security,testsuite
+    """
+    def setUp(self):
+        '''
+        Install the basic packages to support ima-evm-utils
+        '''
+        # Check for basic utilities
+        smm = SoftwareManager()
+        deps = ['gcc', 'make', 'autoconf', 'automake', 'asciidoc', 'libtool']
+        detected_distro = distro.detect()
+        if detected_distro.name in ['rhel', 'fedora', 'centos', 'redhat']:
+            deps.extend(["keyutils-libs-devel", "libxslt", "openssl-devel",
+                         "tpm2-tss-devel"])
+        elif 'SuSE' in detected_distro.name:
+            deps.extend(['attr', 'docbook-xsl-stylesheets', 'keyutils-devel',
+                         'libattr-devel', 'libxslt-tools', 'openssl-devel',
+                         'tpm2-0-tss-devel'])
+        else:
+            self.cancel("Install the package for audit supported\
+                      by %s" % detected_distro.name)
```

Comment:
@nasastry I don't think cancel is required here. or we need to tweak the cancel message like distro is not supported for this test.
which package is not installed here? 
can you please re-check this?

Reply:
@Naresh-ibm For the other test cases also have this `cancel` .. changed the message and sending other version. Thanks!!